v2/paths/prompts-id.yaml

```
@@ -0,0 +1,69 @@
+# SPDX-License-Identifier: GPL-3.0
+# SPDX-FileCopyrightText: Canonical Ltd
+
+get:
+  tags:
+    - Interfaces
+    - OpenAccess
+    - Prompts
+    - Synchronous
+  summary: Retrieve a prompt by ID
+  description: |-
+    Retrieves the details of a single prompt specified by its unique ID.
+  operationId: getPromptById
+  security: []
+  parameters:
+    - name: id
+      in: path
+      required: true
+      description: The unique identifier of the prompt to retrieve.
+      schema:
+        type: string
+    - $ref: '../components/parameters/UserId.yaml'
+  responses:
+    200:
+      description: The details of the requested prompt.
+      content:
+        application/json:
+          schema:
+            $ref: '../components/schemas/Prompt.yaml'
+    404:
+      $ref: '../components/responses/NotFound.yaml'
+
+post:
+  tags:
+    - Interfaces
+    - OpenAccess
+    - Prompts
+    - Synchronous
+  summary: Reply to a prompt
+  description: |-
+    Submit a reply (allow or deny) to an outstanding prompt, potentially
+    creating a new rule.
+  operationId: replyToPrompt
+  security: []
+  parameters:
+    - name: id
+      in: path
+      required: true
+      description: The unique identifier of the prompt to which to reply.
+      schema:
+        type: string
+    - $ref: '../components/parameters/UserId.yaml'
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: '../components/schemas/PromptRequest.yaml'
```

I'd prefer to rename this to `PromptReply`, since it's the reply contents, and "prompt request" means something different internally. I do get the reasoning though, it's the request contents to POST to the prompt endpoint.

```suggestion
          $ref: '../components/schemas/PromptReply.yaml'
```